### DISCUSSION TITLE
feat (extras): Laravel

### DISCUSSION THREAD

**Discussion by muneebusmani** [LazyVim/LazyVim#3562] [Q&A]
im working on laravel extra but there is an issue integrating my current laravel setup for an extra, my nvim config directory has these folders:
![2024-06-10-15:02:56-screenshot](https://github.com/LazyVim/LazyVim/assets/120111082/bb7ccef3-7abe-46d0-a8c7-08a94d80b7f9)
queries/blade
after/queries/html
after/queries/php

how can i implement these tree into LazyVim as an extra, if i added them directly, should i just copy paste them into LazyVim directory and do a pr or something else, i am doing this method because tree sitter doesnt officially support blade files

**Comment by muneebusmani** [comment]
any ideas ? @dpetka2001

**Reply by dpetka2001** [reply]
I'm sorry, but I really don't know. I'm not that well-versed with treesitter myself. These directories are part of the user configuration, so maybe add documentation about what the users should do in their personal config? You should try pinging Folke, as he might have a better idea about how to best integrate this if it's possible at all. Sorry I can't be of more help.

**Comment by muneebusmani** [comment]
@folke can you give any ideas ?

**Comment by ronald2wing** [comment]
Here is an extra you can use https://github.com/LazyVim/LazyVim/pull/6830
